ui/src/app/job-details/panels/panels.component.html

```diff
@@ -37,13 +37,31 @@
       </ng-container>
     </b></p>
     <p id="submitted" *ngIf="job.submission">
-      Submitted: {{ job.submission | jmShortDateTime }}
+      Submitted:
+      <clr-tooltip>
+        <span clrTooltipTrigger> {{ job.submission | jmShortDateTime }}</span>
+        <clr-tooltip-content clrPosition="top-right" clrSize="xs" *clrIfOpen>
+          <span>{{ job.submission | jmFullDateTime }}</span>
+        </clr-tooltip-content>
+      </clr-tooltip>
     </p>
     <p id="started" *ngIf="job.start">
-      Started: {{ job.start | jmShortDateTime }}
+      Started:
+      <clr-tooltip>
```

Comment:
opt: A lot of repetition - this could be extracted to a shared component

Reply:
@calbach Good idea – done.